### PR TITLE
Remove files excluded from linting

### PR DESCRIPTION
**Pull request by cdosborn**
## Description
Lint moar files. Now there is only one file excluded from linting `settings/local.py`.

Nearly every error fixed here is an undefined variable error. Most of the changes don't honestly matter, because the code before 100% didn't work. But there are changes here that can affect working code paths :mag:
## Checklist before merging Pull Requests
- [ ] New test(s) included to reproduce the bug/verify the feature
- [ ] Documentation created/updated at [Example link to documentation](https://example.test/doc#new_section) to give context to the feature
- [ ] If creating/modifying DB models which will contain secrets or sensitive information, PR to [clank](https://github.com/cyverse/clank) updating sanitation queries in `roles/sanitary-sql-access/templates/sanitize-dump.sh.j2`
- [ ] Reviewed and approved by at least one other contributor.
- [ ] If necessary, include a snippet in CHANGELOG.md
- [ ] New variables supported in Clank
- [ ] New variables committed to secrets repos

### CODE REVIEW

File: prospector_profile.yaml [hunk 1]
```diff
@@ -1,15 +1,5 @@
...
-  - core/models/project.py
-  - core/models/t.py
-  - scripts/admin_clean_provider_machine.py
-  - service/accounts/aws.py
-  - service/accounts/eucalyptus.py
-  - service/instance.py
-  - api/v1/views/identity_membership.py
-  - api/v1/views/meta.py
   - settings/local.py
-  - settings/secrets.py
```
Comment (julianpistorius) [inline prospector_profile.yaml]: Can we please also exclude `secrets.py`? Really anything under `atmosphere/settings`.

**Comment by julianpistorius** [comment]
This warms my heart. :) Thanks @cdosborn!

File: prospector_profile.yaml (hunk 1 above)
Comment (cdosborn) [reply prospector_profile.yaml]: That's removed in a later commit

Comment (cdosborn) [reply prospector_profile.yaml]: I have a plan to remove `settings/local.py`, also, but will require more work. Essentially I want one file of variables `settings/__init__.py` templated by clank.

**Comment by cdosborn** [comment]
In the case that the author introduces a :bug:. Should be easy to revert a single commit.

File: prospector_profile.yaml (hunk 1 above)
Comment (julianpistorius) [reply prospector_profile.yaml]: Ahah! OK. I'll stand by.

**Comment by julianpistorius** [comment]
@cdosborn Is this ready to merge?

**Comment by cdosborn** [comment]
It should probably go into master.

**Comment by cdosborn** [comment]
My hunch is that master is behind v30 and that master would need to be rolled forward before this pr could go in. Let me check.

**Comment by julianpistorius** [comment]
@cdosborn What did you find? How can I help get this in?

**Pull request merged**